granule.py

```diff
@@ -0,0 +1,391 @@
+"""
+This module contains functions that will segment images for 
+granule detection.
+"""
+
+import os
+import matplotlib.pyplot as plt
+import matplotlib.lines as lines
+import matplotlib.patheffects as mpe
+import matplotlib.colors as colors
+import numpy as np
+import scipy.ndimage as sndi
+import skimage
+import sunpy
+import sunpy.map
+from sklearn.cluster import KMeans as KMeans
+
+__all__ = [
+    "segment",
+    "get_threshold",
+    "trim_intergranules",
+    "mark_faculae",
+    "segment_kmeans",
+    "cross_correlation",
+]
+
+def segment(id, data_map, skimage_method, res, plot_intermed=True,
+            mark_dim_centers=False, out_dir='output/'):
+
+    """
+    Segment optical image of the solar photosphere into tri-value maps
+    with 0 = intergranule, 0.5 = faculae, 1 = granule.
+    ----------
+    Parameters:

```

Comment:
So the style here should be of the form:

```

    Parameters
    ----------

	id : `str`
        Identifying name of map to be segmented, for file naming.
```
This would need to be done for all of the documentation strings in this PR.

Reply:
I will go ahead and fix this!